docs/src/development/linux.md

```diff
@@ -70,10 +70,31 @@ cargo test --workspace
 
 Zed has basic support for both modes. The mode is selected at runtime. If you're on wayland and want to run in X11 mode, you can set `WAYLAND_DISPLAY='' cargo run` to do so.
 
+## Notes for packaging Zed
+
+Thank you for taking on the task of packaging Zed! There are a few things to note to make things work smoothly:
+
+* Zed is a *very* fast moving project. We typically ship 2-3 [releases per week](/docs/development/releases) to fix user-reported issues and major new features. Please make sure that you are packaging in a way where updates will be available to users within a few hours or days.
```

Comment:
> Please make sure that you are packaging in a way where updates will be available to users within a few hours or days.

It doesn’t work that way; you can’t tell distros how often to update the package and definitely not _within a few hours_. Package maintainers do their best to ship important security fixes within a few hours, if possible, not feature updates of random applications. Zed is just one of many thousands of packages.